.github/renovate.json5

```diff
@@ -0,0 +1,66 @@
+{
+  "extends": [
+    "config:recommended"
+  ],
+  "pip-compile": {
+    "enabled": true,
+    "fileMatch": ["(^|/)requirements\\.in$"]
+  },
+  // pip-compile handles requirements.txt updates.
+  "pip_requirements": {
+    "enabled": false
+  },
+  "constraints": {
+    "python": "~=3.11.0"
+  },
+  "prConcurrentLimit":10,
+  "stabilityDays":7,
+  "vulnerabilityAlerts":{
+     "labels":[
+       "type:security"
+     ],
+     "stabilityDays":0
+  },
+  "labels": ["dependencies"],
+  "kubernetes": {
+    "fileMatch": ["\\.yaml$"],
+    "ignorePaths": [
+      "kubernetes-manifests/**"
```

Comment:
**Issue:**
```suggestion
      "release/**",
      "kustomize/base/**"
```

* Most files inside `release/` folder and the `kustomize/base/` only ever get modified when we make new release of Online Boutique. 
* Example release PR: https://github.com/GoogleCloudPlatform/microservices-demo/pull/1844/files
* _[/docs/releasing/](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/docs/releasing)_ contains more info about what happens during releases. But the tl;dr: 
- we run a script called `make-release.sh`, which 
    - builds Online Boutique locally
    - pushes those built images to gcr.io/google-samples
    - [copies the Kubernetes manifests inside `/kubernetes-manifests/`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/release/v0.8.0/docs/releasing/make-release-artifacts.sh#L97) into `release/` folder and the `kustomize/base/` .

Reply:
Update: according to [PR 1325](https://github.com/GoogleCloudPlatform/microservices-demo/pull/1325), the reason we ignored `kubernetes-manifests/**` is to avoid getting too many Renovate PRs. Sounds reasonable. 
But let's try configuring Renovate to work on `kubernetes-manifests/**`.

Reply:
Nim, thanks for all the context!
I think there's more we can do to help with Renovate for releases, but i'll need to understand the process a bit more before moving forward.
I'll move this discussion into my renovate proposal, and we can discuss further there.